.gitmodules

```diff
@@ -363,6 +363,9 @@
 [submodule "contrib/idna"]
 	path = contrib/idna
 	url = https://github.com/ada-url/idna.git
+[submodule "contrib/cbor"]
```

Comment:
This code does not have a LICENSE and can't be used here.

Reply:
LICENSE in the src folder

Reply:
:+1: